pants

```diff
@@ -15,6 +15,10 @@ set -eou pipefail
 
 PYTHON_BIN_NAME="${PYTHON:-unspecified}"
 
+PANTS_INI=${PANTS_INI:-pants.ini}
+PANTS_TOML=${PANTS_TOML:-pants.toml}
+PANTS_BIN_NAME="${PANTS_BIN_NAME:-$0}"
```

Comment:
It would help to explain in a brief comment "Else, default to the name of this script."

Reply:
I assume the reader knows what `$0` means. I don't like comments that just recapitulate the logic, they are superfluous.